TI-reports/2024/2024-Q4-BEST-WG.md

```diff
@@ -0,0 +1,225 @@
+
+We seek to fortify the open-source ecosystem by championing and embedding best security practices, thereby creating a digital environment where both developers and users can trust and rely on open-source solutions without hesitation.
+
+The BEST Working Group continues to curate and create artifacts tailored towards (open source) developers and open source software consumers illustrating secure development best practices.  This is done through the combination of training collateral, best practices guides, and educational awareness.
+
+- We envision a world where software developers can easily IDENTIFY good practices, requirements and tools that help them create and maintain secure world-class software, helping foster a community where security knowledge is shared and amplified.
+- We seek to provide means to LEARN techniques of writing and identifying secure software using methods best suited to learners of all types.
+- We desire to provide tools to help developers ADOPT these good practices seamlessly into their daily work.
+
+<img align="top" src="https://github.com/ossf/wg-best-practices-os-developers/blob/main/img/OpenSSF%20Dev%20Best%20Practices%20Projects%20Relations.png">
+
+The group continues to be active and is working on several simultaneous projects aligned with our Mission & Vision.  Attendance generally is down, and several former key contributors no longer attend meetings.
+
+
+### Key Resources
+
+- Best Practices for OSS For Software Developers [link](https://best.openssf.org/developers)
+- Best Practices Guides [link](https://openssf.org/resources/guides/)
+- Secure Software Development Fundamentals Course [LFD121](https://training.linuxfoundation.org/training/developing-secure-software-lfd121/)
+- Security Toolbelt - ARCHIVED - [link](https://github.com/ossf/toolbelt)
+
+
+### Sub-groups
+
+- Guides - [link](https://github.com/ossf/wg-best-practices-os-developers/tree/main/docs)
+- EDU.SIG - [link](https://github.com/ossf/education/)
+- Memory Safety SIG - [link](https://github.com/ossf/Memory-Safety)
+- OpenSSF Best Practices Badge - [link](https://www.bestpractices.dev/)
+- Scorecard - [link](https://github.com/ossf/scorecard)
+- Best Practices Badge and Developing Secure Software (LFD121) course - [link](https://github.com/ossf/secure-sw-dev-fundamentals)
+- Security Baseline - [link](https://github.com/ossf/security-baseline)
+- Web (with SWAG) - [link](https://github.com/w3c-cg/swag/)
+
+### Leads
+
+- WG - Avishay Balter & Georg Kunz  
+- OpenSSF Best Practices Badge and Developing Secure Software (LFD121) course - David A. Wheeler
+- Compiler Hardening Guides - Thomas Nyman & Georg Kunz
+- EDU SIG - CRob & Dave Russo
+- Memory Safety SIG - Nell Shamrell-Harrignton & Avishay Balter
+- Python Hardening Guide - Helge Wehder & Georg Kunz
+- Scorecard - Laurent Simon & Stephen Augustus
+- Security Baseline - Eddie Knight
+- WebDev Sec BP - Daniel Appelquist
+
+
+## Activity
+
+### Best Practices Badge
```

Comment:
Hello @david-a-wheeler. Can you please add a status update for the Best Practices Badge, LFD121, and the EDU.SIG? Thanks!

Reply:
Will do!